docker-compose.yml

```diff
@@ -137,9 +137,10 @@ services:
       - HYPERSWITCH_SERVER_URL=${HYPERSWITCH_SERVER_URL:-http://hyperswitch-server:8080}
       - HYPERSWITCH_CLIENT_URL=${HYPERSWITCH_CLIENT_URL:-http://localhost:9050}
       - SELF_SERVER_URL=${SELF_SERVER_URL:-http://localhost:5252}
-    depends_on:
-      hyperswitch-server:
-        condition: service_started
+      - SDK_ENV=${SDK_ENV:-local}
+      - ENV_SDK_URL=${ENV_SDK_URL:-http://localhost:9050/HyperLoader.js}
+      - ENV_BACKEND_URL=${ENV_BACKEND_URL:-http://localhost:8080}
+      - ENV_LOGGING_URL=${ENV_LOGGING_URL:-http://localhost:3100}
```

Comment:
I believe you picked this up from the issue that was mentioned in the fix, but that change hasn't been merged to `main` branch yet,

either use the older SDK variable keys or pull that branch instead of main in your dockerfile